app/assets/javascripts/app/views/people.coffee

```diff
@@ -1,16 +1,16 @@
 PersonView = Backbone.View.extend {
   className: 'card'
   template: _.template "<img class='image' src='<%= imagePath %>'></img>" +
-    "<span class='name'><%= name %></span>" +
-    "<span class='title'><%= title %></span>" +
-    "<span class='bio'><%= bio %></span>"
-
+    "<a class='name virtual' id='<%= _url.split('/')[2].split('.')[0] %>' href='#'>" +
+      "<%= name %>" +
+    "</a>" +
+    "<span class='title'><%= title %></span>"
   render: ->
     @$el.html @template(@model.toJSON())
```

Comment:
FYI
toJSON should not be used for rendering to a template, its purpose is to serialize the data when sending it to the server.

https://github.com/jashkenas/backbone/pull/2135

--

Rather you should be doing a clone of the attributes and passing that to the template render fn
https://github.com/marionettejs/backbone.marionette/issues/1476


:sparkles: